packages/thumbprint-react/components/Popover/index.module.scss

```diff
@@ -0,0 +1,77 @@
+@import '~@thumbtack/thumbprint-tokens/dist/scss/_index';
+@import '~@thumbtack/thumbprint-scss/mixins';
+
+.root {
+    background: $tp-color__white;
+    box-shadow: 0px 2px 10px rgba(0, 0, 0, 0.2);
+    padding: $tp-space__3 $tp-space__4;
```

Comment:
Can you check with @jonpkerwin on what he wants for the padding? The values here are different than they are in Figma.

Reply:
I'll ping him. The ones in figma were not all standard tokens units so I used the closest options.